data/core/units.cfg

```diff
@@ -195,19 +195,19 @@ The eastern provinces of Wesnoth, known as the Clan Homelands, have a geography
         {HUMAN_NAMES}
     [/race]
 
-    # wmllint: general spellings Khalifate Khalid Khaiyal Qanas Saree Faris
+    # wmllint: general spellings Dunefolk Khalid Khaiyal Qanas Saree Faris
     # wmllint: general spellings Qatif-al-nar Rami
     # wmllint: general spelling liminal
     [race]
-        id=khalifate
-        male_name= _"race^Khalifate Human"
-        female_name= _"race+female^Khalifate Human"
-        plural_name= _"race+plural^Khalifate"
+        id=dunefolk
```

Comment:
This is also kinda problematic. What's really needed is for the engine to see `race=khalifate` and instead assign `race=dunefolk`... worst-case scenario, I suppose you could duplicate the entire race definition (just temporarily, until 1.14).